lib/daru/index.rb

```diff
@@ -330,5 +342,12 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def display input_indexes
```

Comment:
The way this function is being called, it does not _display_ anything. It merely creates another index which conforms to a new requirement. The name you have given would have been legit if it were actually PRINTING something to STDOUT.

`conform` would be a better name.